ddp-workspace/projects/ddp-dsm-ui/src/app/file-download/file-download.component.ts

```diff
@@ -21,9 +21,8 @@ export class FileDownloadComponent {
     this.downloadFileEvent.emit(file);
   }
 
-  getNiceDateFormat( uploadedAt: string ): string {
-    return new DatePipe( 'en-US' ).transform( uploadedAt, Utils.DATE_STRING_IN_CVS_WITH_TIME );
+  getUtilStatic(): typeof Utils {
```

Comment:
a bit easier to understand and debug would be have a property, like
```
import {Utils} from '../utils/utils';
...
export class FileDownloadComponent {
  Utils = Utils;
}
```
and use it in the template like
`<td>{{Utils.getNiceDateFormat(file.scannedAt)}}</td>`